jsk_naoqi_robot/naoqieus/naoqi-interface.l

```diff
@@ -254,10 +254,11 @@
        (setq wait t))
      ;; set volume and language
      (when volume
-       (send self :set-master-volume (round (* volume 100))))
+       (unless (send self :set-master-volume (round (* volume 100)))
+         (error "Invalid volume argument")))
      (when lang
        (unless (send self :set-language lang)
-         (ros::ros-error ":speak :lang ~A must be English or Japanese" lang)))
+	 (error "Invalid language argument")))
```

Comment:
Please fix tab indent.

Reply:
Thank you very much for your review.
I fixed tab indent.